lib/rubocop/cop/layout/space_inside_hash_literal_braces.rb

```diff
@@ -36,6 +36,32 @@ module Layout
       #
       #   # good
       #   h = { a: { b: 2 }}
+      #
+      #
+      # @example EnforcedStyleForEmptyBraces: no_space (default)
+      #   # The `no_space` EnforcedStyleForEmptyBraces style enforces that
+      #   # empty hash braces do not contain spaces.
+      #
+      #   # bad
+      #   foo = { }
+      #   bar = {    }
+      #
+      #   # good
+      #   foo = {}
+      #   bar = {}
+      #
+      # @example EnforcedStyleForEmptyBraces: space
+      #   # The `space` EnforcedStyleForEmptyBraces style enforces that
+      #   # empty hash braces contain space.
+      #
+      #   # bad
+      #   foo = {}
+      #
+      #   # good
+      #   foo = { }
+      #   foo = {  }
+      #   foo = {     }
```

Comment:
This documentation highlights the difference in `EnforcedStyleForEmptyBraces/Brackets: space` configuration for these 4 cops. #5304 raises this issue, but it appears that writeup could now use some more detail.